### PR TITLE
Refactor imports and format code using isort and black

### PR DESCRIPTION
**Pull request by VitalinaMykoliuk**

### CODE REVIEW

File: tracker/utils.py [hunk 1]
```diff
@@ -190,27 +194,28 @@ def get_issues_without_pull_requests(
...
-                    lambda issue: issue.get("state") == "open"
-                    and not issue.get("assignee")
-                    and not issue.get("draft")
-                    and not issue.get("pull_request"),
-                    issues,
-                )
+        available_issues = list(
+            filter(
+                lambda issue: issue.get("state") == "open"
+                and not issue.get("assignee")
```
Comment (iamnovichek) [inline tracker/utils.py]: Please move 
```
                and not issue.get("assignee")
                and not issue.get("draft")
                and not issue.get("pull_request"),
```
to `not any()`

**Review by iamnovichek — CHANGES_REQUESTED** [review]

File: tracker/telegram/bot.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 from aiogram.utils.deep_linking import create_start_link
 from aiogram.utils.keyboard import ReplyKeyboardBuilder, ReplyKeyboardMarkup
 from dotenv import load_dotenv
+from .templates import TEMPLATES
```
Comment (djeck1432) [inline tracker/telegram/bot.py]: don't use absolute import, use `app.templates` example

File: tracker/telegram/bot.py [hunk 3]
```diff
@@ -60,9 +63,11 @@ async def start_message(message: Message) -> None:
     :param message: Message that starts the bot.
     :return: None
     """
+    message_text = TEMPLATES["greeting"].substitute(
```
Comment (djeck1432) [inline tracker/telegram/bot.py]: Create dataclass for TemplateNames: with `greeting` `repo_header` and etc use them. If you have everything in one place it's easier to maintaince in the future

File: tracker/utils.py [hunk 4]
```diff
@@ -69,21 +69,20 @@ def check_issue_assignment_events(issue: dict) -> dict:
...
-            for event in events:
-                if event.get("event") == "assigned":
-                    assignment_info["assignee"] = event.get("assignee", dict()).get(
-                        "login", str()
-                    )
-                    assignment_info["assigned_at"] = event.get("created_at", str())
-
-            return assignment_info
+        events = response.json()
+        assignment_info = {
```
Comment (djeck1432) [inline tracker/utils.py]: Use defaultdict if you need to return here empty dict
`event.get("assignee", dict())`

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by VitalinaMykoliuk** [comment]
added screenshots from telegram bot
Screenshot_32
Screenshot_33
Screenshot_34
Screenshot_35

**Comment by VitalinaMykoliuk** [comment]
Added new
Screenshot_42
Screenshot_43
 screenshots from telegram bot

**Review by iamnovichek — APPROVED** [review]

**Pull request merged**